README.md

```diff
@@ -1,6 +1,14 @@
 # rmw_connext
 Implementation of the ROS Middleware (rmw) Interface using [RTI's Connext DDS](https://www.rti.com). 
 
+**DEPRECATION NOTICE:** [rmw_connextdds](https://github.com/ros2/rmw_connextdds)
```

Comment:
No sure if "Deprecation Notice" is the correct term. Please feel free to suggest better wording, thanks!

Reply:
Migration Notice?

Reply:
Deprecation Notice is fine with me.

Reply:
I'll leave it as "Deprecation Notice"